woocommerce/woocommerce
# Update incentives logic - minor addition

Related to https://github.com/Automattic/woocommerce-payments-server/issues/3531

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:
- Attach incentive ID to WCPay connect URL.

### How to test the changes in this Pull Request:

- Setup a JN site with this PR branch (using Live Branches)
- Meet Switch/Action incentive eligibility, more details in #38689.
- Go to the Incentive page and click "Install WooPayments for free"
- Open your browser dev tools network tab and check that you get a request to `<your_site>/wpc-admin/admin.php?wcpay-connect=1&promo=wcpay-promo-2023-etc&_wpnonce=somenone` with a `302` response code (redirect)
- You should have WCPay installed and get redirected to the WCPay KYC
